pulpcore/cli/common/generic.py

```diff
@@ -217,11 +219,12 @@ def load_json_callback(
 
 
 def create_content_json_callback(
-    content_ctx: Type[PulpContentContext], schema: s.Schema = None
+    content_ctx: Optional[Type[PulpContentContext]] = None, schema: s.Schema = None
```

Comment:
Are we breaking any interface if we properly rename this parameter to `ctx_class`?